app/assets/stylesheets/application.css

```diff
@@ -19,10 +19,6 @@ body {
   font-size: 18px;
 }
 
-h3 {
```

Comment:
This was overriding the defaults from skeleton, and meant that `h3` and `h5` were the same font size.